docs/user/alerting/alerting-troubleshooting.asciidoc

```diff
@@ -199,3 +199,17 @@ In addition to the above methods, broadly used the next approaches and common is
 include::troubleshooting/alerting-common-issues.asciidoc[]
 include::troubleshooting/event-log-index.asciidoc[]
 include::troubleshooting/testing-connectors.asciidoc[]
+
+[discrete]
+[[alerting-kibana-disable]]
+==== Temporarily disable
+
+Cluster performance may become degraded from excessive or expensive {kib} rules. If 
+{kib} is sluggish or unresponsive, you may wish to temporarily reduce load to the
+Task Manager by updating <<task-manager-settings-kb.html|settings>>:
+
+[source, txt]
+--------------------------------------------------
+xpack.task_manager.max_workers: 1
```

Comment:
We have another way of doing this, via `xpack.task_manager.unsafe.exclude_task_types`, described in this FAQ/issue - https://github.com/elastic/response-ops-team/issues/31 

Would we want to pull some of this in here?

Reply:
That setting is no longer mentioned in the linked issue, therefore I think this suggestion is no longer valid.

Reply:
Ah, makes sense.  I know we give out the settings mentioned in the linked issue, so let's not add this undocumented one ...